package.json

```diff
@@ -2,18 +2,17 @@
   "name": "hexo-fs",
   "version": "3.1.0",
   "description": "File system module for Hexo.",
-  "main": "lib/fs.js",
+  "main": "./dist/fs.js",
```

Comment:
After execute `npm run build` the source code will be generated in the `dist` folder.